go-controller/pkg/cni/bandwidth_test.go

```diff
@@ -0,0 +1,280 @@
+				{"Command", []string{"string", "string", "string", "string", "string", "string", "string", "string", "string"}, []interface{}{mockCmd}},
+				{"Command", []string{"string", "string", "string", "string", "string", "string", "string", "string", "string"}, []interface{}{mockCmd}},
+				{"Command", []string{"string", "string", "string", "string", "string", "string", "string", "string", "string"}, []interface{}{mockCmd}},
+			},
+			onRetArgsCmdList: []ovntest.TestifyMockHelper{
+				{"CombinedOutput", []string{}, []interface{}{[]byte{1}, nil}},
+				{"CombinedOutput", []string{}, []interface{}{nil, nil}},
+				{"CombinedOutput", []string{}, []interface{}{[]byte{1}, nil}},
+				{"CombinedOutput", []string{}, []interface{}{nil, fmt.Errorf("failed to run ovsDestroy")}},
+			},
+			runnerInstance: mockKexecIface,
+		},
+		{
+			desc:        "Positive test code path",
+			expectedErr: nil,
+			onRetArgsKexecIface: []ovntest.TestifyMockHelper{
+				{"Command", []string{"string", "string", "string", "string", "string", "string", "string", "string", "string"}, []interface{}{mockCmd}},
+				{"Command", []string{"string", "string", "string", "string", "string", "string", "string", "string", "string"}, []interface{}{mockCmd}},
+				{"Command", []string{"string", "string", "string", "string", "string", "string", "string", "string", "string"}, []interface{}{mockCmd}},
+				{"Command", []string{"string", "string", "string", "string", "string", "string", "string", "string", "string"}, []interface{}{mockCmd}},
+			},
+			onRetArgsCmdList: []ovntest.TestifyMockHelper{
+				{"CombinedOutput", []string{}, []interface{}{[]byte{1}, nil}},
+				{"CombinedOutput", []string{}, []interface{}{nil, nil}},
+				{"CombinedOutput", []string{}, []interface{}{[]byte{1}, nil}},
+				{"CombinedOutput", []string{}, []interface{}{nil, nil}},
+			},
+			runnerInstance: mockKexecIface,
+		},
+	}
+	for i, tc := range tests {
+		t.Run(fmt.Sprintf("%d:%s", i, tc.desc), func(t *testing.T) {
+			if tc.onRetArgsKexecIface != nil {
+				for _, item := range tc.onRetArgsKexecIface {
+					ifaceCall := mockKexecIface.On(item.OnCallMethodName)
+					for _, arg := range item.OnCallMethodArgType {
+						ifaceCall.Arguments = append(ifaceCall.Arguments, mock.AnythingOfType(arg))
+					}
+					for _, ret := range item.RetArgList {
+						ifaceCall.ReturnArguments = append(ifaceCall.ReturnArguments, ret)
+					}
+					ifaceCall.Once()
+				}
+			}
+			if tc.onRetArgsCmdList != nil {
+				for _, item := range tc.onRetArgsCmdList {
+					mockCall := mockCmd.On(item.OnCallMethodName)
+					for _, arg := range item.OnCallMethodArgType {
+						mockCall.Arguments = append(mockCall.Arguments, mock.AnythingOfType(arg))
+					}
+					for _, ret := range item.RetArgList {
+						mockCall.ReturnArguments = append(mockCall.ReturnArguments, ret)
+					}
+					mockCall.Once()
+				}
+			}
+			runner = tc.runnerInstance
```

Comment:
it would be helpful to others if you can add a comment above L123 to note that runner is defined in pkg/cni/ovs.go file